rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -895,10 +895,9 @@ namespace rsx
 
 	namespace nv308a
 	{
-		template<u32 index>
 		struct color
 		{
-			static void impl(thread* rsx, u32 /*reg*/, u32 /*arg*/)
+			static void core(thread* rsx, u32 index, u32 x700minus_idx)
```

Comment:
```suggestion
			static void core(thread* rsx, u32 rsx, u32 /*arg*/)
```
Simply use tge reg argument, index = reg - NV308A_COLOR.

Reply:
Okay, that should work too.